rhel/run.sh

```diff
@@ -45,6 +45,21 @@ if [ -z "${INSTANA_DOWNLOAD_KEY}" ]; then
   INSTANA_DOWNLOAD_KEY="${INSTANA_AGENT_KEY}"
 fi
 
+# Take over Agent Proxy variables if no Repository Proxy is enabled
+case ${INSTANA_REPOSITORY_PROXY_ENABLED} in
+  y|Y|yes|Yes|YES|1|true)
```

Comment:
@mdonkers what about `enabled` ? :trollface:

Reply:
`ENABLED=enabled` ?? Nah! It's no JavaScript :-P

Reply:
🤣